README.md

```diff
@@ -81,6 +81,7 @@ Besides the above, there are many possible options within `config.yml` for confi
     - `offer_draw_moves`: The absolute value of the evaluation has to be less than or equal to `offer_draw_score` for `offer_draw_moves` amount of moves for the bot to offer/accept draw.
     - `offer_draw_pieces`: The bot only offers/accepts draws if the position has less than or equal to `offer_draw_pieces` pieces.
 - `online_moves`: This section gives your bot access to various online resources for choosing moves like opening books and endgame tablebases. This can be a supplement or a replacement for chess databases stored on your computer. There are three sections that correspond to three different online databases:
+    - `max_out_of_book_moves`: Stop using online opening books after they don't have a move for `max_out_of_book_moves` positions. Doesn't apply for the online endgame tablebases.
```

Comment:
This line should be moved below the DB listings (since the previous sentence says that it's going to name them) and above the DB-specific configurations (above `- Configurations common to all`.